src/test/suite/language-tools/java.test.ts

```diff
@@ -126,6 +126,17 @@ suite('Java Language Tools', () => {
       status: TestStatus.Failed,
     })
     assert.strictEqual(result, undefined)
+
```

Comment:
a couple more fuzz test cases for that regex would be good to have

Reply:
Updated - also updated regex this handles unexpected combinations of brackets.